src/main/java/com/rakovets/course/java/core/practice/oop/classesandobjects/task_3/model/Rectangle.java

```diff
@@ -0,0 +1,36 @@
+package com.rakovets.course.java.core.practice.oop.classesandobjects.task_3.model;
+
+import com.rakovets.course.java.core.practice.oop.classesandobjects.task_2.model.Point;
+
+public class Rectangle {
+    private Point FirstPoint;
+    private Point SecondPoint;
```

Comment:
Those are wrong names for variables.
```suggestion
    private Point firstPoint;
    private Point secondPoint;
```

Reply:
Ok, i got it)